frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java

```diff
@@ -21,18 +21,15 @@ public final class MacheteFileUtils {
   private MacheteFileUtils() {}
 
   public static String getSampleMacheteFileContents() {
-    // We're deliberately using \n rather than `System.lineSeparator()` here
-    // since it turned out that even on Windows (which generally uses \r\n) IntelliJ expects \n in Color Settings code sample
-    // (and probably in Code Style as well).
-    @SuppressWarnings("regexp") String NL = "\n";
+    return """
```

Comment:
Still looks as expected:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/3383210/222982578-2c817354-7877-4c06-accb-c3da608dfc44.png">

Reply:
(this is in Code Style settings)

Reply:
great <3